Description: Add the provider version for compatibility with plainbox >= 0.5
Author: Sylvain Pineau <sylvain.pineau@canonical.com>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/plainbox-resources.provider
+++ b/plainbox-resources.provider
@@ -2,3 +2,4 @@
 name = 2013.com.canonical:plainbox-resources
 location = /usr/lib/plainbox-providers-1/plainbox-resources/
 description = PlainBox resources provider
+version = 1.0
